readthedocs/projects/managers.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class HTMLFileManager(models.Manager):
+
+    def get_queryset(self):
+        return super(HTMLFileManager, self).get_queryset().filter(is_html=True)
```

Comment:
Can't this just do `filter(filename__endswith='html')` instead of adding additional state to the model?

Reply:
Yeah. It can be done. I thought it would be slower, so I added another state.
But I think performance is not a issue here. So I am good to filter by name.